modules/icinga/manifests/client.pp

```diff
@@ -36,10 +36,19 @@
     ],
   }
 
-  @@icinga::host { $::fqdn:
-    hostalias    => $::fqdn,
-    address      => $::ipaddress,
-    display_name => $::fqdn_short,
+  if ($::vdc == 'licensify') or ($::vdc == 'efg') or ($::vdc =~ /^*_dr$/) {
+    @@icinga::host { $::fqdn:
+      hostalias    => $::fqdn,
+      address      => $::ipaddress,
+      display_name => $::fqdn_short,
+      parents      => "vpn_gateway_${::vdc}",
+    }
+  } else {
+    @@icinga::host { $::fqdn:
+      hostalias    => $::fqdn,
+      address      => $::ipaddress,
+      display_name => $::fqdn_short,
+    }
```

Comment:
Rather than repeating the `icinga::host` block twice, it would be preferable to always set a `parents` parameter but leave it empty or undefined for EFG and Licensify DR hosts.